<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit" />


<title>REV vs REV_TAX vs DMSO</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.18/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.9.4.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-1.57.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-1.57.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Sara_REV</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">REV vs REV_TAX vs DMSO</h1>
<h4 class="author">Fabio Bedin | MS-Unit</h4>
<h4 class="date">12 luglio 2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-07-14
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Sara_REV/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210712code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210712)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210712code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210712)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong77b9e27"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 77b9e27 </a>
</p>
</div>
<div id="strongRepositoryversionstrong77b9e27" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 77b9e27. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  code/custom_functions.R
    Untracked:  data/expdesign.tsv
    Untracked:  data/proteinGroups_DMSO_REV_REV_TAX.txt

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/REV_REVTAX_DMSO.Rmd</code>) and HTML (<code>docs/REV_REVTAX_DMSO.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
77b9e27
</td>
<td>
FabioBedin
</td>
<td>
2021-07-14
</td>
<td>
upload project analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>knitr::opts_chunk$set(fig.align = &quot;center&quot;, warning=FALSE, message=FALSE, fig.height=10, fig.width=10)
library(&quot;MBQN&quot;)
library(&quot;dplyr&quot;)
library(&quot;tidyr&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;tibble&quot;)
library(&quot;ggplot2&quot;)
library(&quot;enrichR&quot;)
library(&quot;DT&quot;)
library(&quot;stringr&quot;)
library(&quot;patchwork&quot;)
library(&quot;here&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;openxlsx&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(&quot;data/proteinGroups_DMSO_REV_REV_TAX.txt&quot;), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)

data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot; &amp; data$Peptides &gt;= 2,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<div id="pca-all-samples" class="section level1">
<h1>PCA all samples</h1>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign.tsv&quot;), header = T, stringsAsFactors = F)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

data_filt &lt;- filter_missval(data_se, thr = 0)

data_diff &lt;- test_diff_BH(data_filt, type = &quot;all&quot;)

dep &lt;- add_rejections(data_diff, alpha = 0.05, lfc = 1)

results&lt;-get_results(dep)

plot_pca(dep, x = 1, y = 2, n = 500, point_size = 4, indicate = c(&quot;condition&quot;, &quot;replicate&quot;))</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/unnamed-chunk-2-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>cond_1 &lt;- &quot;REV.&quot;

cond_2 &lt;- &quot;DMSO&quot;

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)</code></pre>
</div>
<div id="analysis" class="section level1 tabset tabset-fade">
<h1><strong>Analysis</strong></h1>
<div id="rev._vs_dmso" class="section level2 tabset tabset-fade">
<h2><strong>REV._vs_DMSO</strong></h2>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(cond_1,cond_2)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

data_filt_Unique &lt;- filter_missval(data_se, thr = 0)

data_filt &lt;- filter_by_grouping(data_se, thr = 0)</code></pre>
<div id="proteins" class="section level3">
<h3>Proteins</h3>
<pre class="r"><code>plot_numbers_lables(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/proteins_1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="normalization" class="section level3">
<h3>normalization</h3>
<pre class="r"><code>plot_normalization(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/normalization_1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="frequency" class="section level3">
<h3>frequency</h3>
<pre class="r"><code>plot_frequency(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/frequency_1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot" class="section level3 tabset tabset-fade">
<h3>upset plot</h3>
<div id="rev." class="section level4">
<h4>REV.</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_1.1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="dmso" class="section level4">
<h4>DMSO</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_1.2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="both" class="section level4">
<h4>Both</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 6,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;turquoise3&quot;, &quot;coral2&quot;), each = 3),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_1.3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="dep-analysis" class="section level3 tabset tabset-fade">
<h3>DEP analysis</h3>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = 0.05, lfc = 1)

results&lt;-get_results(dep)</code></pre>
<p>Significant: 0</p>
<div id="pca" class="section level4">
<h4>PCA</h4>
<pre class="r"><code>plot_pca(dep, x = 1, y = 2, n = 500, point_size = 4, indicate = c(&quot;condition&quot;, &quot;replicate&quot;))</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/PCA_1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T,label_size=5,adjusted = F)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/volcano_1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="unique" class="section level3">
<h3>Unique</h3>
<pre class="r"><code>unique_pair(data_filt_Unique, conditions = conditions)</code></pre>
<div id="htmlwidget-026fb29cbaf759d52142" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-026fb29cbaf759d52142">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68"],["ABCB8","ACOT2","ATP7A","CDKN2A.1","CIAO1","COG1","COG4","DDX11","DIEXF","FAM45A","GGA1","HMGCR","IDH3A","IMPACT","ISOC1","JAK1","KCTD2","KIAA1033","LGALSL","MPO","MPP5","MRPL14","MRPS10","MRPS34","NARF","NELFA","NPRL3","OGFOD3","PCNP","PDP1","PFDN5","PIGU","PNKP","PPP1R7","PRKAR2B","PVR","RADIL","RALY","RANBP10","RBM8A","RGPD5","RPRD2","RPS6KA4","RXRB","SART3","SF3A3","SGK3","SLAIN2","SLC25A32","SMN1","SMYD4","SPEN","SPTAN1","SRSF3","STAMBP","STAU2","STK24","STK33","TECPR2","TNPO2","TOPBP1","TTC12","TUBA4A","USP19","USP54","VTI1A","XPR1","ZNF326"],[false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false],[true,true,true,true,true,true,true,true,true,true,true,true,false,true,false,true,true,true,false,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,false,true,false,true,true,true,true,true]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>unique_REV.<\/th>\n      <th>unique_DMSO<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cc-unique" class="section level3">
<h3>CC Unique</h3>
<pre class="r"><code>tab_unique &lt;- unique_pair(data_filt_Unique, conditions = conditions, table = T)

GO_name &lt;- tab_unique %&gt;% filter(unique_REV.) %&gt;% pull(name)

GO &lt;- gost(query = GO_name, organism = &quot;hsapiens&quot;, sources = &quot;GO:CC&quot;, ordered_query = F, significant = F)

gostplot(GO, capped = TRUE, interactive = TRUE)</code></pre>
<div id="htmlwidget-452e0b38c6afa4b4d9c8" style="width:960px;height:960px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-452e0b38c6afa4b4d9c8">{"x":{"data":[{"x":[145.75767730763,9.50774025494114,95.2470224563399,3.82140444054785,96.5353329142884,96.4909084157385,96.4464839171885,92.670401540443,92.5371280447931,84.7184163000024,81.9640973899056,71.7020382248677,66.5487963930738,66.5043718945239,66.4599473959739,66.2378249032242,66.1934004046742,65.9268534133745,65.8824289148246,96.5797574128384,96.6241819113883,96.6686064099383,96.7130309084882,173.256441910048,169.969029017352,169.924604518802,169.880180020252,168.591869562303,168.547445063753,157.308046930617,60.6847625844807,144.069546362732,139.049578026588,123.811975023956,122.790211557307,122.745787058757,117.459271731314,112.839123882119,96.7574554070382,141.093104959886,56.9086802077352,56.0201902367362,49.9340339353934,19.2811299359296,19.1478564402798,19.1034319417298,19.0590074431799,17.4153009968318,16.3046885330831,15.8160190490337,15.0608025736846,14.5721330896352,14.3055860983355,12.6618796519874,12.6174551534375,8.08615630134282,6.93111933904418,6.88669484049423,6.70899684629444,4.93201690429654,19.9030729156289,177.921014257792,20.1251954083786,20.4805913967782,49.8451849382935,45.5360085789486,41.9376241964029,41.8931996978529,39.7608237674554,39.7163992689055,39.1833052863061,39.1388807877562,38.2503908167572,24.834192254673,24.3899472691736,23.8124287880243,22.8350898199254,22.4352693329759,22.2575713387761,20.658289390978,20.5694403938781,20.2140444054785,182.141341620037],"y":[0.603368462955076,0.126934225807782,0.00233895249500652,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0],"text":["GO:0097013 (5) <br> phagocytic vesicle lumen <br> 2.492e-01","GO:0001673 (15) <br> male germ cell nucleus <br> 7.466e-01","GO:0043073 (20) <br> germ cell nucleus <br> 9.946e-01","GO:0000228 (237) <br> nuclear chromosome <br> 1.000e+00","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 1.000e+00","GO:0043227 (13201) <br> membrane-bounded organelle <br> 1.000e+00","GO:0043226 (14027) <br> organelle <br> 1.000e+00","GO:0042582 (154) <br> azurophil granule <br> 1.000e+00","GO:0042579 (146) <br> microbody <br> 1.000e+00","GO:0035578 (91) <br> azurophil granule lumen <br> 1.000e+00","GO:0034774 (320) <br> secretory granule lumen <br> 1.000e+00","GO:0032991 (5541) <br> protein-containing complex <br> 1.000e+00","GO:0031983 (326) <br> vesicle lumen <br> 1.000e+00","GO:0031982 (4058) <br> vesicle <br> 1.000e+00","GO:0031981 (4453) <br> nuclear lumen <br> 1.000e+00","GO:0031975 (1244) <br> envelope <br> 1.000e+00","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 1.000e+00","GO:0031967 (1244) <br> organelle envelope <br> 1.000e+00","GO:0031966 (750) <br> mitochondrial membrane <br> 1.000e+00","GO:0043229 (13117) <br> intracellular organelle <br> 1.000e+00","GO:0043230 (2265) <br> extracellular organelle <br> 1.000e+00","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 1.000e+00","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 1.000e+00","GO:0110165 (18814) <br> cellular anatomical entity <br> 1.000e+00","GO:0099513 (766) <br> polymeric cytoskeletal fiber <br> 1.000e+00","GO:0099512 (997) <br> supramolecular fiber <br> 1.000e+00","GO:0099503 (1027) <br> secretory vesicle <br> 1.000e+00","GO:0099081 (1005) <br> supramolecular polymer <br> 1.000e+00","GO:0099080 (1325) <br> supramolecular complex <br> 1.000e+00","GO:0097708 (2438) <br> intracellular vesicle <br> 1.000e+00","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 1.000e+00","GO:0090575 (183) <br> RNA polymerase II transcription regulator complex <br> 1.000e+00","GO:0071682 (22) <br> endocytic vesicle lumen <br> 1.000e+00","GO:0070062 (2178) <br> extracellular exosome <br> 1.000e+00","GO:0070013 (5579) <br> intracellular organelle lumen <br> 1.000e+00","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 1.000e+00","GO:0060205 (324) <br> cytoplasmic vesicle lumen <br> 1.000e+00","GO:0045335 (139) <br> phagocytic vesicle <br> 1.000e+00","GO:0043233 (5579) <br> organelle lumen <br> 1.000e+00","GO:0071944 (6178) <br> cell periphery <br> 1.000e+00","GO:0031224 (5873) <br> intrinsic component of membrane <br> 1.000e+00","GO:0031090 (3644) <br> organelle membrane <br> 1.000e+00","GO:0030141 (857) <br> secretory granule <br> 1.000e+00","GO:0005743 (502) <br> mitochondrial inner membrane <br> 1.000e+00","GO:0005740 (800) <br> mitochondrial envelope <br> 1.000e+00","GO:0005739 (1678) <br> mitochondrion <br> 1.000e+00","GO:0005737 (11951) <br> cytoplasm <br> 1.000e+00","GO:0005694 (1795) <br> chromosome <br> 1.000e+00","GO:0005667 (431) <br> transcription regulator complex <br> 1.000e+00","GO:0005654 (4112) <br> nucleoplasm <br> 1.000e+00","GO:0005634 (7607) <br> nucleus <br> 1.000e+00","GO:0005622 (14808) <br> intracellular anatomical structure <br> 1.000e+00","GO:0005615 (3595) <br> extracellular space <br> 1.000e+00","GO:0005576 (4564) <br> extracellular region <br> 1.000e+00","GO:0005575 (18964) <br> cellular_component <br> 1.000e+00","GO:0000922 (168) <br> spindle pole <br> 1.000e+00","GO:0000794 (73) <br> condensed nuclear chromosome <br> 1.000e+00","GO:0000793 (239) <br> condensed chromosome <br> 1.000e+00","GO:0000785 (1235) <br> chromatin <br> 1.000e+00","GO:0000323 (723) <br> lytic vacuole <br> 1.000e+00","GO:0005759 (482) <br> mitochondrial matrix <br> 1.000e+00","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 1.000e+00","GO:0005764 (723) <br> lysosome <br> 1.000e+00","GO:0005773 (818) <br> vacuole <br> 1.000e+00","GO:0030139 (335) <br> endocytic vesicle <br> 1.000e+00","GO:0019866 (564) <br> organelle inner membrane <br> 1.000e+00","GO:0016605 (102) <br> PML body <br> 1.000e+00","GO:0016604 (816) <br> nuclear body <br> 1.000e+00","GO:0016021 (5718) <br> integral component of membrane <br> 1.000e+00","GO:0016020 (9829) <br> membrane <br> 1.000e+00","GO:0015630 (1313) <br> microtubule cytoskeleton <br> 1.000e+00","GO:0015629 (517) <br> actin cytoskeleton <br> 1.000e+00","GO:0012505 (4651) <br> endomembrane system <br> 1.000e+00","GO:0005886 (5680) <br> plasma membrane <br> 1.000e+00","GO:0005874 (435) <br> microtubule <br> 1.000e+00","GO:0005856 (2339) <br> cytoskeleton <br> 1.000e+00","GO:0005829 (5361) <br> cytosol <br> 1.000e+00","GO:0005819 (390) <br> spindle <br> 1.000e+00","GO:0005815 (815) <br> microtubule organizing center <br> 1.000e+00","GO:0005777 (146) <br> peroxisome <br> 1.000e+00","GO:0005775 (174) <br> vacuolar lumen <br> 1.000e+00","GO:0005766 (154) <br> primary lysosome <br> 1.000e+00","GO:1903561 (2263) <br> extracellular vesicle <br> 1.000e+00"],"key":["GO:0097013","GO:0001673","GO:0043073","GO:0000228","GO:0043228","GO:0043227","GO:0043226","GO:0042582","GO:0042579","GO:0035578","GO:0034774","GO:0032991","GO:0031983","GO:0031982","GO:0031981","GO:0031975","GO:0031974","GO:0031967","GO:0031966","GO:0043229","GO:0043230","GO:0043231","GO:0043232","GO:0110165","GO:0099513","GO:0099512","GO:0099503","GO:0099081","GO:0099080","GO:0097708","GO:0031410","GO:0090575","GO:0071682","GO:0070062","GO:0070013","GO:0065010","GO:0060205","GO:0045335","GO:0043233","GO:0071944","GO:0031224","GO:0031090","GO:0030141","GO:0005743","GO:0005740","GO:0005739","GO:0005737","GO:0005694","GO:0005667","GO:0005654","GO:0005634","GO:0005622","GO:0005615","GO:0005576","GO:0005575","GO:0000922","GO:0000794","GO:0000793","GO:0000785","GO:0000323","GO:0005759","GO:0140513","GO:0005764","GO:0005773","GO:0030139","GO:0019866","GO:0016605","GO:0016604","GO:0016021","GO:0016020","GO:0015630","GO:0015629","GO:0012505","GO:0005886","GO:0005874","GO:0005856","GO:0005829","GO:0005819","GO:0005815","GO:0005777","GO:0005775","GO:0005766","GO:1903561"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":[0.3,0.3,0.3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"size":[3.77952755905512,10.6794384793225,11.530371380266,16.7106846766617,21.1065376980832,22.2571486793644,22.3281816653382,15.9669234838839,15.8717089827301,14.9926529098301,17.2043828589237,21.2094525319788,17.234331696469,20.8178366753038,20.9355557744668,19.2412882830615,21.217946716426,19.2412882830615,18.5151217974708,22.2496616018047,20.0594063051708,22.149467645466,21.1045752249039,22.6680579579949,18.546128464597,18.9279382110163,18.9702841150749,18.9393654128159,19.3294362411644,20.1570755154885,20.1543591908086,16.2699008434032,11.7923848464024,20.0071923232852,21.217946716426,20.0594063051708,17.2244178275065,15.7833464644507,21.217946716426,21.3442086680117,21.2816415799842,20.6804368134123,18.7099376935951,17.9124590505498,18.6097176553623,19.6551392843042,22.1401261988699,19.7468678585021,17.6767018733908,20.8346394095116,21.5990392966823,22.3913687515746,20.6630716149315,20.966623935922,22.6771653543307,16.120649456977,14.5583522451083,16.7247579286407,19.231109533402,18.4611109276066,17.8499898638447,19.2322444942848,18.4611109276066,18.6421914076905,17.2781168573742,18.089885189333,15.2110332592996,18.6386221642431,21.2484979483869,21.9079715449555,19.3167538465386,17.9575268973122,20.9904129457737,21.24022548979,17.6910976904051,20.10213853464,21.1683503233887,17.5199639323906,18.6368339382872,15.8717089827301,16.1821073330082,15.9669234838839,20.0582305127984],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData026fb29c","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,200.044414535666],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":45.251350032552,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:CC"],"tickvals":[101.022207267833],"categoryorder":"array","categoryarray":["GO:CC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"45d414914123":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"45d452964114":{"x":{},"xend":{},"y":{},"yend":{}},"45d467d4769c":{"x":{},"y":{}},"45d46cea782":{"yintercept":{}}},"cur_data":"45d414914123","visdat":{"45d414914123":["function (y) ","x"],"45d452964114":["function (y) ","x"],"45d467d4769c":["function (y) ","x"],"45d46cea782":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData026fb29c"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>cond_1 &lt;- &quot;REV.&quot;

cond_2 &lt;- &quot;REVTAX&quot;

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)</code></pre>
</div>
</div>
<div id="rev._vs_revtax" class="section level2 tabset tabset-fade">
<h2><strong>REV._vs_REVTAX</strong></h2>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(cond_1,cond_2)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

data_filt_Unique &lt;- filter_missval(data_se, thr = 0)

data_filt &lt;- filter_by_grouping(data_se, thr = 0)</code></pre>
<div id="proteins-1" class="section level3">
<h3>Proteins</h3>
<pre class="r"><code>plot_numbers_lables(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/proteins_2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="normalization-1" class="section level3">
<h3>normalization</h3>
<pre class="r"><code>plot_normalization(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/normalization_2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="frequency-1" class="section level3">
<h3>frequency</h3>
<pre class="r"><code>plot_frequency(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/frequency_2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-1" class="section level3 tabset tabset-fade">
<h3>upset plot</h3>
<div id="rev.-1" class="section level4">
<h4>REV.</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_2.1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="revtax" class="section level4">
<h4>REVTAX</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_2.2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="both-1" class="section level4">
<h4>Both</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 6,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;turquoise3&quot;, &quot;coral2&quot;), each = 3),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_2.3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="dep-analysis-1" class="section level3 tabset tabset-fade">
<h3>DEP analysis</h3>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = 0.05, lfc = 1)

results&lt;-get_results(dep)</code></pre>
<p>Significant: 0</p>
<div id="pca-1" class="section level4">
<h4>PCA</h4>
<pre class="r"><code>plot_pca(dep, x = 1, y = 2, n = 500, point_size = 4, indicate = c(&quot;condition&quot;, &quot;replicate&quot;))</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/PCA_2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-1" class="section level4">
<h4>Volcano</h4>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T,label_size=5,adjusted = F)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/volcano_2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="unique-1" class="section level3">
<h3>Unique</h3>
<pre class="r"><code>unique_pair(data_filt_Unique, conditions = conditions)</code></pre>
<div id="htmlwidget-48def2144534479d6da6" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-48def2144534479d6da6">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51"],["ACTG1","AKT1","AMBRA1","APPL2","ARHGAP17","ATL1","ATP7A","C5orf51","COTL1","COX15","CRYZ","DCPS","DDX49","GCAT","GNAQ","HADH","IPO13","ISOC1","JAK1","KIF13A","MAGEA1","MEN1","MPO","MPP5","MRPL14","NIT1","NUDT19","OTUD5","PICK1","PIGG","POLR3B","PVR","QSER1","R3HCC1","RANBP10","RBM12","RRN3","SART3","SMARCA1","SMYD4","SRA1","SRSF1","STK33","VWA8","WASF3","WDR45B","XPR1","YIPF6","ZC3H14","ZC3H7B","ZNF326"],[true,true,false,false,false,false,false,true,true,true,false,true,true,true,true,true,false,true,false,false,true,true,true,false,false,true,true,false,false,false,false,false,false,true,false,false,false,false,false,false,true,false,false,false,false,true,false,false,false,false,false],[false,false,true,true,true,true,true,false,false,false,true,false,false,false,false,false,true,false,true,true,false,false,false,true,true,false,false,true,true,true,true,true,true,false,true,true,true,true,true,true,false,true,true,true,true,false,true,true,true,true,true]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>unique_REV.<\/th>\n      <th>unique_REVTAX<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cc-unique-1" class="section level3">
<h3>CC Unique</h3>
<pre class="r"><code>tab_unique &lt;- unique_pair(data_filt_Unique, conditions = conditions, table = T)

GO_name &lt;- tab_unique %&gt;% pull(name)

GO &lt;- gost(query = GO_name, organism = &quot;hsapiens&quot;, sources = &quot;GO:CC&quot;, ordered_query = F, significant = F)

gostplot(GO, capped = TRUE, interactive = TRUE)</code></pre>
<div id="htmlwidget-4150ddbfc19706c0a3bb" style="width:960px;height:960px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4150ddbfc19706c0a3bb">{"x":{"data":[{"x":[96.4909084157385,14.5721330896352,96.6686064099383,96.5797574128384,112.839123882119,96.4464839171885,49.8451849382935,15.0608025736846,19.0590074431799,56.7754067120853,83.341256844954,70.6802747582189,175.655364831745,17.282027501182,96.5353329142884,96.7130309084882,96.7574554070382,97.4682473838373,96.3132104215387,96.2687859229887,96.2243614244388,97.6015208794872,100.089292798284,96.6241819113883,2.75521647534911,96.1799369258889,115.771140786416,115.326895800916,114.882650815417,110.662323453172,110.529049957522,110.484625458972,104.398469157629,110.395776461872,109.640559986523,107.996853540175,105.642355117028,105.420232624278,105.331383627178,104.798289644579,110.218078467672,104.753865146029,94.9360509664903,95.0693244621401,66.5932208916237,66.6376453901737,66.7709188858235,67.7038333553724,67.7482578539224,68.9921438133209,70.635850259669,71.3022177379182,71.3466422364681,71.7020382248677,73.7455651581653,76.7220065610118,77.9214680218604,80.8534849261569,96.1355124273389,81.9640973899056,84.7184163000024,85.3403592797016,85.3847837782516,87.3394617144493,87.6948577028489,89.1608661549971,89.2052906535471,91.781911569444,92.0928830592937,92.5371280447931,92.670401540443,94.4473814824409,117.459271731314,94.9804754650402,83.1635588507542,118.658733192162,122.790211557307,122.745787058757,169.880180020252,168.591869562303,168.547445063753,166.503918130456,166.237371139156,164.682513689908,169.924604518802,164.504815695708,164.282693202958,164.149419707309,164.016146211659,163.571901226159,163.527476727609,163.17208073921,164.460391197158,163.12765624066,169.969029017352,170.368849504301,183.518501075086,183.163105086686,182.985407092486,182.585586605537,182.141341620037,179.69799419979,170.235576008651,179.076051220091,177.921014257792,174.322629875247,174.278205376697,173.434139904248,173.256441910048,170.768669991251,178.054287753442,163.08323174211,163.03880724356,162.99438274501,142.026019429435,141.093104959886,140.604435475836,139.049578026588,132.297054246996,131.763960264397,143.625301377233,128.565396368801,126.210897945653,124.789313992055,124.744889493505,124.034097516706,123.811975023956,66.5487963930738,127.454783905052,145.66882831053,145.75767730763,145.80210180618,162.94995824646,162.416864263861,162.328015266761,161.128553805912,161.084129307362,160.151214837813,159.706969852314,157.707867417566,157.619018420466,157.574593921916,157.308046930617,152.910021574172,151.088617133624,150.777645643774,146.20192229313,119.147402676212,66.5043718945239,66.1934004046742,66.2378249032242,21.5467793619769,21.457930364877,21.4135058663271,21.3690813677771,21.1913833735773,21.1469588750274,21.5912038605269,20.9248363822777,20.658289390978,20.613864892428,20.5694403938781,20.5250158953281,20.4805913967782,20.3917423996783,20.8804118837277,20.3473179011283,21.7244773561767,22.1687223416762,25.8115312227719,25.5894087300222,25.5449842314722,24.834192254673,24.7453432575732,24.4343717677235,21.7689018547267,24.3899472691736,23.8124287880243,23.2793348054249,23.0127878141252,22.8350898199254,22.4352693329759,22.2575713387761,24.2566737735237,25.8559557213218,20.3028934025784,20.1696199069286,12.6174551534375,10.3518057273901,10.1741077331904,8.26385429554261,6.70899684629444,6.5757233506446,12.6618796519874,5.95378037094533,4.93201690429654,4.84316790719664,4.75431891009675,3.06618796519874,2.844065472449,2.79964097389906,5.6428088810957,20.2140444054785,14.3055860983355,15.8160190490337,20.1251954083786,20.0363464112787,19.9919219127288,19.9030729156289,19.4144034315794,19.2811299359296,15.1052270722345,19.1922809388297,19.1034319417298,18.7480359533302,17.4153009968318,16.8377825156825,16.3046885330831,16.2602640345332,19.1478564402798,66.4599473959739,26.3002007068213,26.5223231995711,56.9086802077352,56.0201902367362,55.1317002657373,53.4435693208393,53.0437488338898,52.9993243353398,56.9975292048351,52.0664098657909,51.6221648802914,51.5333158831915,51.3556178889917,50.8225239063924,50.7780994078424,50.1561564281432,51.8887118715911,49.9340339353934,57.041953703385,57.9748681729339,188.849440901079,65.9712779119245,65.9268534133745,65.8824289148246,65.8380044162747,65.7491554191748,57.3529251932347,65.3049104336753,63.0392610076279,60.6847625844807,58.6856601497331,58.6412356511831,58.1525661671337,58.0192926714839,65.0383634423756,26.3890497039212,49.8896094368435,49.7119114426437,39.2277297848561,39.1833052863061,39.1388807877562,39.0056072921063,38.5613623066068,38.2948153153072,39.7163992689055,38.2503908167572,37.0953538544586,36.9620803588087,36.8288068631589,28.4770011357687,26.8777191879706,26.566747698121,37.2730518486584,49.7563359411936,39.7608237674554,40.2939177500548,49.6674869440937,49.6230624455438,47.5795355122462,47.5351110136963,47.4906865151463,47.3574130194965,40.2494932515049,45.6692820745985,45.5804330774986,45.5360085789486,42.0264731935028,41.8931996978529,41.5822282080033,40.5160402428045,45.6248575760485,199.289198060317],"y":[2.91691708781356,2.86288142989471,2.82708885213369,2.16743725360232,2.12924628285521,1.80050579655707,1.26016238800962,1.09084072819083,0.977100352355492,0.963097260656886,0.509620419713364,0.273982694850698,0.178798473644978,0.178798473644978,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0],"text":["GO:0043227 (13201) <br> membrane-bounded organelle <br> 1.211e-03","GO:0005622 (14808) <br> intracellular anatomical structure <br> 1.371e-03","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 1.489e-03","GO:0043229 (13117) <br> intracellular organelle <br> 6.801e-03","GO:0045335 (139) <br> phagocytic vesicle <br> 7.426e-03","GO:0043226 (14027) <br> organelle <br> 1.583e-02","GO:0030139 (335) <br> endocytic vesicle <br> 5.493e-02","GO:0005634 (7607) <br> nucleus <br> 8.113e-02","GO:0005737 (11951) <br> cytoplasm <br> 1.054e-01","GO:0031209 (12) <br> SCAR complex <br> 1.089e-01","GO:0035145 (20) <br> exon-exon junction complex <br> 3.093e-01","GO:0032588 (98) <br> trans-Golgi network membrane <br> 5.321e-01","GO:0120220 (1) <br> basal body patch <br> 6.625e-01","GO:0005691 (1) <br> U6atac snRNP <br> 6.625e-01","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 1.000e+00","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 1.000e+00","GO:0043233 (5579) <br> organelle lumen <br> 1.000e+00","GO:0043292 (242) <br> contractile fiber <br> 1.000e+00","GO:0043220 (11) <br> Schmidt-Lanterman incisure <br> 1.000e+00","GO:0043219 (4) <br> lateral loop <br> 1.000e+00","GO:0043218 (14) <br> compact myelin <br> 1.000e+00","GO:0043296 (146) <br> apical junction complex <br> 1.000e+00","GO:0043679 (128) <br> axon terminus <br> 1.000e+00","GO:0043230 (2265) <br> extracellular organelle <br> 1.000e+00","GO:0000137 (26) <br> Golgi cis cisterna <br> 1.000e+00","GO:0043209 (49) <br> myelin sheath <br> 1.000e+00","GO:0055029 (117) <br> nuclear DNA-directed RNA polymerase complex <br> 1.000e+00","GO:0048770 (109) <br> pigment granule <br> 1.000e+00","GO:0048471 (745) <br> perinuclear region of cytoplasm <br> 1.000e+00","GO:0045202 (1348) <br> synapse <br> 1.000e+00","GO:0045178 (268) <br> basal part of cell <br> 1.000e+00","GO:0045177 (433) <br> apical part of cell <br> 1.000e+00","GO:0044297 (595) <br> cell body <br> 1.000e+00","GO:0045171 (76) <br> intercellular bridge <br> 1.000e+00","GO:0045009 (18) <br> chitosome <br> 1.000e+00","GO:0044609 (2) <br> DBIRD complex <br> 1.000e+00","GO:0044391 (191) <br> ribosomal subunit <br> 1.000e+00","GO:0044354 (8) <br> macropinosome <br> 1.000e+00","GO:0044352 (8) <br> pinosome <br> 1.000e+00","GO:0044306 (146) <br> neuron projection terminus <br> 1.000e+00","GO:0045121 (339) <br> membrane raft <br> 1.000e+00","GO:0044305 (20) <br> calyx of Held <br> 1.000e+00","GO:0042995 (2337) <br> cell projection <br> 1.000e+00","GO:0043025 (523) <br> neuronal cell body <br> 1.000e+00","GO:0031984 (1764) <br> organelle subcompartment <br> 1.000e+00","GO:0031985 (117) <br> Golgi cisterna <br> 1.000e+00","GO:0032009 (12) <br> early phagosome <br> 1.000e+00","GO:0032153 (72) <br> cell division site <br> 1.000e+00","GO:0032154 (54) <br> cleavage furrow <br> 1.000e+00","GO:0032279 (345) <br> asymmetric synapse <br> 1.000e+00","GO:0032587 (99) <br> ruffle membrane <br> 1.000e+00","GO:0032838 (225) <br> plasma membrane bounded cell projection cytoplasm <br> 1.000e+00","GO:0032839 (34) <br> dendrite cytoplasm <br> 1.000e+00","GO:0032991 (5541) <br> protein-containing complex <br> 1.000e+00","GO:0033162 (18) <br> melanosome membrane <br> 1.000e+00","GO:0034045 (15) <br> phagophore assembly site membrane <br> 1.000e+00","GO:0034399 (132) <br> nuclear periphery <br> 1.000e+00","GO:0034708 (101) <br> methyltransferase complex <br> 1.000e+00","GO:0043204 (160) <br> perikaryon <br> 1.000e+00","GO:0034774 (320) <br> secretory granule lumen <br> 1.000e+00","GO:0035578 (91) <br> azurophil granule lumen <br> 1.000e+00","GO:0035749 (6) <br> myelin sheath adaxonal region <br> 1.000e+00","GO:0035770 (252) <br> ribonucleoprotein granule <br> 1.000e+00","GO:0036064 (162) <br> ciliary basal body <br> 1.000e+00","GO:0036186 (2) <br> early phagosome membrane <br> 1.000e+00","GO:0036464 (238) <br> cytoplasmic ribonucleoprotein granule <br> 1.000e+00","GO:0036477 (875) <br> somatodendritic compartment <br> 1.000e+00","GO:0042175 (1154) <br> nuclear outer membrane-endoplasmic reticulum membrane network <br> 1.000e+00","GO:0042470 (109) <br> melanosome <br> 1.000e+00","GO:0042579 (146) <br> microbody <br> 1.000e+00","GO:0042582 (154) <br> azurophil granule <br> 1.000e+00","GO:0042734 (148) <br> presynaptic membrane <br> 1.000e+00","GO:0060205 (324) <br> cytoplasmic vesicle lumen <br> 1.000e+00","GO:0043005 (1384) <br> neuron projection <br> 1.000e+00","GO:0035097 (72) <br> histone methyltransferase complex <br> 1.000e+00","GO:0061574 (4) <br> ASAP complex <br> 1.000e+00","GO:0070013 (5579) <br> intracellular organelle lumen <br> 1.000e+00","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 1.000e+00","GO:0099503 (1027) <br> secretory vesicle <br> 1.000e+00","GO:0099081 (1005) <br> supramolecular polymer <br> 1.000e+00","GO:0099080 (1325) <br> supramolecular complex <br> 1.000e+00","GO:0098984 (371) <br> neuron to neuron synapse <br> 1.000e+00","GO:0098978 (354) <br> glutamatergic synapse <br> 1.000e+00","GO:0098862 (164) <br> cluster of actin-based cell projections <br> 1.000e+00","GO:0099512 (997) <br> supramolecular fiber <br> 1.000e+00","GO:0098858 (217) <br> actin-based cell projection <br> 1.000e+00","GO:0098852 (395) <br> lytic vacuole membrane <br> 1.000e+00","GO:0098845 (15) <br> postsynaptic endosome <br> 1.000e+00","GO:0098842 (3) <br> postsynaptic early endosome <br> 1.000e+00","GO:0098827 (1137) <br> endoplasmic reticulum subcompartment <br> 1.000e+00","GO:0098826 (5) <br> endoplasmic reticulum tubular network membrane <br> 1.000e+00","GO:0098798 (272) <br> mitochondrial protein-containing complex <br> 1.000e+00","GO:0098857 (339) <br> membrane microdomain <br> 1.000e+00","GO:0098797 (696) <br> plasma membrane protein complex <br> 1.000e+00","GO:0099513 (766) <br> polymeric cytoskeletal fiber <br> 1.000e+00","GO:0099572 (363) <br> postsynaptic specialization <br> 1.000e+00","GO:1905360 (36) <br> GTPase complex <br> 1.000e+00","GO:1904949 (115) <br> ATPase complex <br> 1.000e+00","GO:1904813 (124) <br> ficolin-1-rich granule lumen <br> 1.000e+00","GO:1904115 (62) <br> axon cytoplasm <br> 1.000e+00","GO:1903561 (2263) <br> extracellular vesicle <br> 1.000e+00","GO:1902494 (1377) <br> catalytic complex <br> 1.000e+00","GO:0099568 (270) <br> cytoplasmic region <br> 1.000e+00","GO:0150034 (307) <br> distal axon <br> 1.000e+00","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 1.000e+00","GO:0120114 (82) <br> Sm-like protein family complex <br> 1.000e+00","GO:0120111 (93) <br> neuron projection cytoplasm <br> 1.000e+00","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 1.000e+00","GO:0110165 (18814) <br> cellular anatomical entity <br> 1.000e+00","GO:0101002 (184) <br> ficolin-1-rich granule <br> 1.000e+00","GO:0140535 (768) <br> intracellular protein-containing complex <br> 1.000e+00","GO:0098796 (1326) <br> membrane protein complex <br> 1.000e+00","GO:0098794 (641) <br> postsynapse <br> 1.000e+00","GO:0098793 (515) <br> presynapse <br> 1.000e+00","GO:0072562 (139) <br> blood microparticle <br> 1.000e+00","GO:0071944 (6178) <br> cell periphery <br> 1.000e+00","GO:0071782 (26) <br> endoplasmic reticulum tubular network <br> 1.000e+00","GO:0071682 (22) <br> endocytic vesicle lumen <br> 1.000e+00","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 1.000e+00","GO:0071001 (3) <br> U4/U6 snRNP <br> 1.000e+00","GO:0090537 (2) <br> CERF complex <br> 1.000e+00","GO:0070603 (79) <br> SWI/SNF superfamily-type complex <br> 1.000e+00","GO:0070382 (228) <br> exocytic vesicle <br> 1.000e+00","GO:0070161 (838) <br> anchoring junction <br> 1.000e+00","GO:0070160 (132) <br> tight junction <br> 1.000e+00","GO:0070069 (34) <br> cytochrome complex <br> 1.000e+00","GO:0070062 (2178) <br> extracellular exosome <br> 1.000e+00","GO:0031983 (326) <br> vesicle lumen <br> 1.000e+00","GO:0070469 (100) <br> respirasome <br> 1.000e+00","GO:0090741 (18) <br> pigment granule membrane <br> 1.000e+00","GO:0097013 (5) <br> phagocytic vesicle lumen <br> 1.000e+00","GO:0097014 (133) <br> ciliary plasm <br> 1.000e+00","GO:0098791 (845) <br> Golgi apparatus subcompartment <br> 1.000e+00","GO:0098687 (348) <br> chromosomal region <br> 1.000e+00","GO:0098685 (84) <br> Schaffer collateral - CA1 synapse <br> 1.000e+00","GO:0098590 (1240) <br> plasma membrane region <br> 1.000e+00","GO:0098588 (2165) <br> bounding membrane of organelle <br> 1.000e+00","GO:0098562 (195) <br> cytoplasmic side of membrane <br> 1.000e+00","GO:0098552 (622) <br> side of membrane <br> 1.000e+00","GO:0097733 (119) <br> photoreceptor cell cilium <br> 1.000e+00","GO:0097731 (130) <br> 9+0 non-motile cilium <br> 1.000e+00","GO:0097730 (164) <br> non-motile cilium <br> 1.000e+00","GO:0097708 (2438) <br> intracellular vesicle <br> 1.000e+00","GO:0097525 (61) <br> spliceosomal snRNP complex <br> 1.000e+00","GO:0097447 (643) <br> dendritic tree <br> 1.000e+00","GO:0097433 (6) <br> dense body <br> 1.000e+00","GO:0097060 (384) <br> synaptic membrane <br> 1.000e+00","GO:0061695 (266) <br> transferase complex, transferring phosphorus-containing groups <br> 1.000e+00","GO:0031982 (4058) <br> vesicle <br> 1.000e+00","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 1.000e+00","GO:0031975 (1244) <br> envelope <br> 1.000e+00","GO:0005797 (11) <br> Golgi medial cisterna <br> 1.000e+00","GO:0005795 (151) <br> Golgi stack <br> 1.000e+00","GO:0005794 (1593) <br> Golgi apparatus <br> 1.000e+00","GO:0005793 (133) <br> endoplasmic reticulum-Golgi intermediate compartment <br> 1.000e+00","GO:0005789 (1132) <br> endoplasmic reticulum membrane <br> 1.000e+00","GO:0005788 (313) <br> endoplasmic reticulum lumen <br> 1.000e+00","GO:0005798 (95) <br> Golgi-associated vesicle <br> 1.000e+00","GO:0005783 (1995) <br> endoplasmic reticulum <br> 1.000e+00","GO:0005777 (146) <br> peroxisome <br> 1.000e+00","GO:0005776 (98) <br> autophagosome <br> 1.000e+00","GO:0005775 (174) <br> vacuolar lumen <br> 1.000e+00","GO:0005774 (450) <br> vacuolar membrane <br> 1.000e+00","GO:0005773 (818) <br> vacuole <br> 1.000e+00","GO:0005770 (285) <br> late endosome <br> 1.000e+00","GO:0005782 (52) <br> peroxisomal matrix <br> 1.000e+00","GO:0005769 (387) <br> early endosome <br> 1.000e+00","GO:0005801 (71) <br> cis-Golgi network <br> 1.000e+00","GO:0005813 (620) <br> centrosome <br> 1.000e+00","GO:0005911 (496) <br> cell-cell junction <br> 1.000e+00","GO:0005903 (109) <br> brush border <br> 1.000e+00","GO:0005902 (89) <br> microvillus <br> 1.000e+00","GO:0005886 (5680) <br> plasma membrane <br> 1.000e+00","GO:0005884 (118) <br> actin filament <br> 1.000e+00","GO:0005875 (164) <br> microtubule associated complex <br> 1.000e+00","GO:0005802 (255) <br> trans-Golgi network <br> 1.000e+00","GO:0005874 (435) <br> microtubule <br> 1.000e+00","GO:0005856 (2339) <br> cytoskeleton <br> 1.000e+00","GO:0005840 (247) <br> ribosome <br> 1.000e+00","GO:0005834 (36) <br> heterotrimeric G-protein complex <br> 1.000e+00","GO:0005829 (5361) <br> cytosol <br> 1.000e+00","GO:0005819 (390) <br> spindle <br> 1.000e+00","GO:0005815 (815) <br> microtubule organizing center <br> 1.000e+00","GO:0005871 (52) <br> kinesin complex <br> 1.000e+00","GO:0005912 (171) <br> adherens junction <br> 1.000e+00","GO:0005768 (993) <br> endosome <br> 1.000e+00","GO:0005765 (395) <br> lysosomal membrane <br> 1.000e+00","GO:0005575 (18964) <br> cellular_component <br> 1.000e+00","GO:0001750 (91) <br> photoreceptor outer segment <br> 1.000e+00","GO:0001726 (180) <br> ruffle <br> 1.000e+00","GO:0000932 (93) <br> P-body <br> 1.000e+00","GO:0000785 (1235) <br> chromatin <br> 1.000e+00","GO:0000781 (160) <br> chromosome, telomeric region <br> 1.000e+00","GO:0005576 (4564) <br> extracellular region <br> 1.000e+00","GO:0000428 (118) <br> DNA-directed RNA polymerase complex <br> 1.000e+00","GO:0000323 (723) <br> lytic vacuole <br> 1.000e+00","GO:0000315 (57) <br> organellar large ribosomal subunit <br> 1.000e+00","GO:0000313 (88) <br> organellar ribosome <br> 1.000e+00","GO:0000151 (299) <br> ubiquitin ligase complex <br> 1.000e+00","GO:0000139 (725) <br> Golgi membrane <br> 1.000e+00","GO:0000138 (11) <br> Golgi trans cisterna <br> 1.000e+00","GO:0000407 (33) <br> phagophore assembly site <br> 1.000e+00","GO:0005766 (154) <br> primary lysosome <br> 1.000e+00","GO:0005615 (3595) <br> extracellular space <br> 1.000e+00","GO:0005654 (4112) <br> nucleoplasm <br> 1.000e+00","GO:0005764 (723) <br> lysosome <br> 1.000e+00","GO:0005762 (57) <br> mitochondrial large ribosomal subunit <br> 1.000e+00","GO:0005761 (88) <br> mitochondrial ribosome <br> 1.000e+00","GO:0005759 (482) <br> mitochondrial matrix <br> 1.000e+00","GO:0005746 (86) <br> mitochondrial respirasome <br> 1.000e+00","GO:0005743 (502) <br> mitochondrial inner membrane <br> 1.000e+00","GO:0005635 (471) <br> nuclear envelope <br> 1.000e+00","GO:0005741 (205) <br> mitochondrial outer membrane <br> 1.000e+00","GO:0005739 (1678) <br> mitochondrion <br> 1.000e+00","GO:0005730 (955) <br> nucleolus <br> 1.000e+00","GO:0005694 (1795) <br> chromosome <br> 1.000e+00","GO:0005681 (192) <br> spliceosomal complex <br> 1.000e+00","GO:0005667 (431) <br> transcription regulator complex <br> 1.000e+00","GO:0005666 (19) <br> RNA polymerase III complex <br> 1.000e+00","GO:0005740 (800) <br> mitochondrial envelope <br> 1.000e+00","GO:0031981 (4453) <br> nuclear lumen <br> 1.000e+00","GO:0005923 (124) <br> bicellular tight junction <br> 1.000e+00","GO:0005929 (674) <br> cilium <br> 1.000e+00","GO:0031224 (5873) <br> intrinsic component of membrane <br> 1.000e+00","GO:0031090 (3644) <br> organelle membrane <br> 1.000e+00","GO:0031010 (12) <br> ISWI-type complex <br> 1.000e+00","GO:0030880 (122) <br> RNA polymerase complex <br> 1.000e+00","GO:0030864 (92) <br> cortical actin cytoskeleton <br> 1.000e+00","GO:0030863 (121) <br> cortical cytoskeleton <br> 1.000e+00","GO:0031226 (1725) <br> intrinsic component of plasma membrane <br> 1.000e+00","GO:0030670 (75) <br> phagocytic vesicle membrane <br> 1.000e+00","GO:0030659 (789) <br> cytoplasmic vesicle membrane <br> 1.000e+00","GO:0030532 (70) <br> small nuclear ribonucleoprotein complex <br> 1.000e+00","GO:0030496 (197) <br> midbody <br> 1.000e+00","GO:0030425 (641) <br> dendrite <br> 1.000e+00","GO:0030424 (661) <br> axon <br> 1.000e+00","GO:0030176 (161) <br> integral component of endoplasmic reticulum membrane <br> 1.000e+00","GO:0030666 (190) <br> endocytic vesicle membrane <br> 1.000e+00","GO:0030141 (857) <br> secretory granule <br> 1.000e+00","GO:0031227 (169) <br> intrinsic component of endoplasmic reticulum membrane <br> 1.000e+00","GO:0031252 (426) <br> cell leading edge <br> 1.000e+00","GO:1990234 (795) <br> transferase complex <br> 1.000e+00","GO:0031968 (230) <br> organelle outer membrane <br> 1.000e+00","GO:0031967 (1244) <br> organelle envelope <br> 1.000e+00","GO:0031966 (750) <br> mitochondrial membrane <br> 1.000e+00","GO:0031965 (301) <br> nuclear membrane <br> 1.000e+00","GO:0031941 (33) <br> filamentous actin <br> 1.000e+00","GO:0031234 (103) <br> extrinsic component of cytoplasmic side of plasma membrane <br> 1.000e+00","GO:0031907 (52) <br> microbody lumen <br> 1.000e+00","GO:0031526 (59) <br> brush border membrane <br> 1.000e+00","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 1.000e+00","GO:0031301 (389) <br> integral component of organelle membrane <br> 1.000e+00","GO:0031300 (419) <br> intrinsic component of organelle membrane <br> 1.000e+00","GO:0031256 (179) <br> leading edge membrane <br> 1.000e+00","GO:0031253 (346) <br> cell projection membrane <br> 1.000e+00","GO:0031901 (167) <br> early endosome membrane <br> 1.000e+00","GO:0005925 (420) <br> focal adhesion <br> 1.000e+00","GO:0030140 (33) <br> trans-Golgi network transport vesicle <br> 1.000e+00","GO:0030135 (296) <br> coated vesicle <br> 1.000e+00","GO:0015934 (119) <br> large ribosomal subunit <br> 1.000e+00","GO:0015630 (1313) <br> microtubule cytoskeleton <br> 1.000e+00","GO:0015629 (517) <br> actin cytoskeleton <br> 1.000e+00","GO:0015030 (57) <br> Cajal body <br> 1.000e+00","GO:0014069 (339) <br> postsynaptic density <br> 1.000e+00","GO:0012506 (1183) <br> vesicle membrane <br> 1.000e+00","GO:0016020 (9829) <br> membrane <br> 1.000e+00","GO:0012505 (4651) <br> endomembrane system <br> 1.000e+00","GO:0009986 (896) <br> cell surface <br> 1.000e+00","GO:0009925 (250) <br> basal plasma membrane <br> 1.000e+00","GO:0009898 (169) <br> cytoplasmic side of plasma membrane <br> 1.000e+00","GO:0008021 (208) <br> synaptic vesicle <br> 1.000e+00","GO:0005938 (326) <br> cell cortex <br> 1.000e+00","GO:0005930 (131) <br> axoneme <br> 1.000e+00","GO:0010008 (520) <br> endosome membrane <br> 1.000e+00","GO:0030136 (194) <br> clathrin-coated vesicle <br> 1.000e+00","GO:0016021 (5718) <br> integral component of membrane <br> 1.000e+00","GO:0016324 (364) <br> apical plasma membrane <br> 1.000e+00","GO:0030134 (90) <br> COPII-coated ER to Golgi transport vesicle <br> 1.000e+00","GO:0030133 (416) <br> transport vesicle <br> 1.000e+00","GO:0030055 (427) <br> cell-substrate junction <br> 1.000e+00","GO:0030054 (2107) <br> cell junction <br> 1.000e+00","GO:0030027 (203) <br> lamellipodium <br> 1.000e+00","GO:0030016 (234) <br> myofibril <br> 1.000e+00","GO:0016323 (225) <br> basolateral plasma membrane <br> 1.000e+00","GO:0019898 (315) <br> extrinsic component of membrane <br> 1.000e+00","GO:0019867 (232) <br> outer membrane <br> 1.000e+00","GO:0019866 (564) <br> organelle inner membrane <br> 1.000e+00","GO:0016607 (413) <br> nuclear speck <br> 1.000e+00","GO:0016604 (816) <br> nuclear body <br> 1.000e+00","GO:0016589 (7) <br> NURF complex <br> 1.000e+00","GO:0016363 (110) <br> nuclear matrix <br> 1.000e+00","GO:0019897 (178) <br> extrinsic component of plasma membrane <br> 1.000e+00","GO:1990904 (694) <br> ribonucleoprotein complex <br> 1.000e+00"],"key":["GO:0043227","GO:0005622","GO:0043231","GO:0043229","GO:0045335","GO:0043226","GO:0030139","GO:0005634","GO:0005737","GO:0031209","GO:0035145","GO:0032588","GO:0120220","GO:0005691","GO:0043228","GO:0043232","GO:0043233","GO:0043292","GO:0043220","GO:0043219","GO:0043218","GO:0043296","GO:0043679","GO:0043230","GO:0000137","GO:0043209","GO:0055029","GO:0048770","GO:0048471","GO:0045202","GO:0045178","GO:0045177","GO:0044297","GO:0045171","GO:0045009","GO:0044609","GO:0044391","GO:0044354","GO:0044352","GO:0044306","GO:0045121","GO:0044305","GO:0042995","GO:0043025","GO:0031984","GO:0031985","GO:0032009","GO:0032153","GO:0032154","GO:0032279","GO:0032587","GO:0032838","GO:0032839","GO:0032991","GO:0033162","GO:0034045","GO:0034399","GO:0034708","GO:0043204","GO:0034774","GO:0035578","GO:0035749","GO:0035770","GO:0036064","GO:0036186","GO:0036464","GO:0036477","GO:0042175","GO:0042470","GO:0042579","GO:0042582","GO:0042734","GO:0060205","GO:0043005","GO:0035097","GO:0061574","GO:0070013","GO:0065010","GO:0099503","GO:0099081","GO:0099080","GO:0098984","GO:0098978","GO:0098862","GO:0099512","GO:0098858","GO:0098852","GO:0098845","GO:0098842","GO:0098827","GO:0098826","GO:0098798","GO:0098857","GO:0098797","GO:0099513","GO:0099572","GO:1905360","GO:1904949","GO:1904813","GO:1904115","GO:1903561","GO:1902494","GO:0099568","GO:0150034","GO:0140513","GO:0120114","GO:0120111","GO:0120025","GO:0110165","GO:0101002","GO:0140535","GO:0098796","GO:0098794","GO:0098793","GO:0072562","GO:0071944","GO:0071782","GO:0071682","GO:0071013","GO:0071001","GO:0090537","GO:0070603","GO:0070382","GO:0070161","GO:0070160","GO:0070069","GO:0070062","GO:0031983","GO:0070469","GO:0090741","GO:0097013","GO:0097014","GO:0098791","GO:0098687","GO:0098685","GO:0098590","GO:0098588","GO:0098562","GO:0098552","GO:0097733","GO:0097731","GO:0097730","GO:0097708","GO:0097525","GO:0097447","GO:0097433","GO:0097060","GO:0061695","GO:0031982","GO:0031974","GO:0031975","GO:0005797","GO:0005795","GO:0005794","GO:0005793","GO:0005789","GO:0005788","GO:0005798","GO:0005783","GO:0005777","GO:0005776","GO:0005775","GO:0005774","GO:0005773","GO:0005770","GO:0005782","GO:0005769","GO:0005801","GO:0005813","GO:0005911","GO:0005903","GO:0005902","GO:0005886","GO:0005884","GO:0005875","GO:0005802","GO:0005874","GO:0005856","GO:0005840","GO:0005834","GO:0005829","GO:0005819","GO:0005815","GO:0005871","GO:0005912","GO:0005768","GO:0005765","GO:0005575","GO:0001750","GO:0001726","GO:0000932","GO:0000785","GO:0000781","GO:0005576","GO:0000428","GO:0000323","GO:0000315","GO:0000313","GO:0000151","GO:0000139","GO:0000138","GO:0000407","GO:0005766","GO:0005615","GO:0005654","GO:0005764","GO:0005762","GO:0005761","GO:0005759","GO:0005746","GO:0005743","GO:0005635","GO:0005741","GO:0005739","GO:0005730","GO:0005694","GO:0005681","GO:0005667","GO:0005666","GO:0005740","GO:0031981","GO:0005923","GO:0005929","GO:0031224","GO:0031090","GO:0031010","GO:0030880","GO:0030864","GO:0030863","GO:0031226","GO:0030670","GO:0030659","GO:0030532","GO:0030496","GO:0030425","GO:0030424","GO:0030176","GO:0030666","GO:0030141","GO:0031227","GO:0031252","GO:1990234","GO:0031968","GO:0031967","GO:0031966","GO:0031965","GO:0031941","GO:0031234","GO:0031907","GO:0031526","GO:0031410","GO:0031301","GO:0031300","GO:0031256","GO:0031253","GO:0031901","GO:0005925","GO:0030140","GO:0030135","GO:0015934","GO:0015630","GO:0015629","GO:0015030","GO:0014069","GO:0012506","GO:0016020","GO:0012505","GO:0009986","GO:0009925","GO:0009898","GO:0008021","GO:0005938","GO:0005930","GO:0010008","GO:0030136","GO:0016021","GO:0016324","GO:0030134","GO:0030133","GO:0030055","GO:0030054","GO:0030027","GO:0030016","GO:0016323","GO:0019898","GO:0019867","GO:0019866","GO:0016607","GO:0016604","GO:0016589","GO:0016363","GO:0019897","GO:1990904"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":[0.8,0.8,0.8,0.8,0.8,0.8,0.3,0.3,0.3,0.3,0.3,0.3,0.3,0.3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"size":[22.3264252195808,22.4383641864847,22.2367192356697,22.3201850649318,17.1548195499605,22.3856494123114,18.298143967892,21.7796081055084,22.2289413688919,13.2710937630066,14.2011261458923,16.6724381791646,3.77952755905512,3.77952755905512,21.3727481966795,21.3711312319661,21.4645991271325,17.8862783424092,13.1034348780947,10.8689453455282,13.5610679933045,17.2212436823674,17.0426137733969,20.5151304912853,14.6479076029226,15.657962724621,16.9192281419459,16.8211511553887,19.263939603674,19.9432483584573,18.0168094072206,18.6150750264817,18.9984666498796,16.3098880006505,14.0162209305128,8.79250295053476,17.5788100584201,12.4622556341903,12.4622556341903,17.2212436823674,18.3129563246243,14.2011261458923,20.548992015662,18.8440542104479,20.2421225987011,16.9192281419459,13.2710937630066,16.2314244319177,15.8053279745816,18.3348227251218,16.6867045005751,17.7923685991039,15.0864964623698,21.4575922354223,14.0162209305128,13.6881024821628,17.0846046613508,16.7147638172736,17.3441685027701,18.2408347685535,16.5678180426824,11.8392984446581,17.9382148021614,17.3607594651419,8.79250295053476,17.864844622288,19.4511014638451,19.7680332004913,16.8211511553887,17.2212436823674,17.2929942112838,17.2395796662231,18.2563981588253,19.9727791267635,16.2314244319177,10.8689453455282,21.4645991271325,20.5151304912853,19.6352925278716,19.6105163746592,19.9239365272364,18.4250322214133,18.366860054541,17.3771269800777,19.6013623632847,17.745456787333,18.5024142596311,13.6881024821628,10.0906276575191,19.7511979989463,11.4182363867305,18.0356600152042,18.3129563246243,19.1840864979984,19.2964481920454,18.3980254244623,15.1777648550946,16.895419938855,16.9991498217807,16.0118000582096,20.5141736049721,19.967101846646,18.0262727377997,18.1887538150907,19.8456741893998,16.4193371991966,16.5985974422729,20.50019667835,22.6695463135235,17.5296740015926,19.2994941394791,19.9247836033461,19.0869091908042,18.8254941646856,17.1548195499605,21.5688278615741,14.6479076029226,14.3656113471999,16.4707645381266,10.0906276575191,8.79250295053476,16.3657708525876,17.8094925358158,19.4010451201652,17.0846046613508,15.0864964623698,20.4726514898635,18.2641017542387,16.7008119247509,14.0162209305128,11.4182363867305,17.0948833273556,19.4106950637145,18.3456016295724,16.4538496314687,19.8493193311577,20.4661491827473,17.6060101019579,19.0512349983319,16.9425908232129,17.0637873175792,17.3771269800777,20.5946659308898,15.9876792754223,19.0905979284939,11.8392984446581,18.4675989226445,18.0072688243436,21.1352513701066,21.4645991271325,19.8529519106499,13.1034348780947,17.2665787007403,20.1294607688252,17.0948833273556,19.74619512479,18.2130832293158,16.6286505174186,20.3770742525653,17.2212436823674,16.6724381791646,17.4558059838392,18.6620554454478,19.3729892715705,18.0949019953727,15.7483038980765,18.477200178829,16.2110466588123,19.0474116732065,18.7801361240538,16.8211511553887,16.536277842633,21.4829800128551,16.9309641680734,17.3771269800777,17.953358393088,18.6207047977596,20.5499166926234,17.912533281388,15.1777648550946,21.4236958550181,18.486721097755,19.3687173999097,15.7483038980765,17.4327344039982,19.5967557732233,18.5024142596311,22.6771653543307,16.5678180426824,17.5006676724288,16.5985974422729,19.8447609312007,17.3441685027701,21.2575544180182,16.9309641680734,19.2288083982256,15.8865533167699,16.5202109991529,18.1554978868918,19.232049357513,13.1034348780947,15.0385342713221,17.2929942112838,21.0082539834581,21.1490529351569,19.2288083982256,15.8865533167699,16.5202109991529,18.745496163536,16.487459483958,18.7946596664582,18.7175072384601,17.6714222468736,20.1869944840899,19.5519741052092,20.2612942389817,17.5856681532369,18.6094170392983,14.1115209894216,19.3471014383054,21.2319995442489,16.9991498217807,19.1462431419456,21.5171615504251,21.0224922535402,13.2710937630066,16.9768337465094,16.5833006324756,16.9655237439507,20.2174863716732,16.2907117380545,19.3309710236804,16.1903456554121,17.6193819587006,19.0869091908042,19.123250680122,17.3524922814642,17.571912525939,19.4270397336233,17.4171052400333,18.5951468122635,19.3397991772088,17.8207723469673,19.8529519106499,19.2717683392144,18.1639018076508,15.0385342713221,16.7422139528449,15.7483038980765,15.9380785666977,20.5924526025259,18.4835563041861,18.5748608988786,17.4933056237393,18.3384269605871,17.4012717563209,18.5777812583696,15.0385342713221,18.1427766693107,16.9425908232129,19.913717949598,18.8301632203654,15.8865533167699,18.3129563246243,19.7961481762761,22.0358650928176,21.27712785025,19.4785105490561,17.9280094542126,17.4171052400333,17.6903669156388,18.2641017542387,17.0742397921166,18.8371303524317,17.5992677528991,21.4898062370426,18.4014363963032,16.5521452913468,18.5660543277138,18.5980153132463,20.4366231496717,17.6586232001651,17.8430140998672,17.7923685991039,18.2210805382519,17.831945559191,18.9345998162058,18.5571787119523,19.3701432319559,12.1788496145115,16.8338388209956,17.4858983432249,19.1806997751779],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedDatabaf759d5","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,200.044414535666],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":45.251350032552,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:CC"],"tickvals":[101.022207267833],"categoryorder":"array","categoryarray":["GO:CC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"45d431ca5e8a":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"45d415a850db":{"x":{},"xend":{},"y":{},"yend":{}},"45d41ae45bf5":{"x":{},"y":{}},"45d45e8e78c4":{"yintercept":{}}},"cur_data":"45d431ca5e8a","visdat":{"45d431ca5e8a":["function (y) ","x"],"45d415a850db":["function (y) ","x"],"45d41ae45bf5":["function (y) ","x"],"45d45e8e78c4":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedDatabaf759d5"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>cond_1 &lt;- &quot;REVTAX&quot;

cond_2 &lt;- &quot;DMSO&quot;

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)</code></pre>
</div>
</div>
<div id="revtax_vs_dmso" class="section level2 tabset tabset-fade">
<h2><strong>REVTAX_vs_DMSO</strong></h2>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign.tsv&quot;), header = T, stringsAsFactors = F)

conditions&lt;-c(cond_1,cond_2)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

data_filt_Unique &lt;- filter_missval(data_se, thr = 0)

data_filt &lt;- filter_by_grouping(data_se, thr = 0)</code></pre>
<div id="proteins-2" class="section level3">
<h3>Proteins</h3>
<pre class="r"><code>plot_numbers_lables(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/proteins_3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="normalization-2" class="section level3">
<h3>normalization</h3>
<pre class="r"><code>plot_normalization(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/normalization_3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="frequency-2" class="section level3">
<h3>frequency</h3>
<pre class="r"><code>plot_frequency(data_filt_Unique)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/frequency_3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-2" class="section level3 tabset tabset-fade">
<h3>upset plot</h3>
<div id="revtax-1" class="section level4">
<h4>REVTAX</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_1)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_3.1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="dmso-1" class="section level4">
<h4>DMSO</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  select(starts_with(cond_2)) %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 3,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_3.2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="both-2" class="section level4">
<h4>Both</h4>
<pre class="r"><code>assay(data_filt_Unique) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = 6,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;turquoise3&quot;, &quot;coral2&quot;), each = 3),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/upset_3.3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="dep-analysis-2" class="section level3 tabset tabset-fade">
<h3>DEP analysis</h3>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = 0.05, lfc = 1)

results&lt;-get_results(dep)</code></pre>
<p>Significant: 0</p>
<div id="pca-2" class="section level4">
<h4>PCA</h4>
<pre class="r"><code>plot_pca(dep, x = 1, y = 2, n = 500, point_size = 4, indicate = c(&quot;condition&quot;, &quot;replicate&quot;))</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/PCA_3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano-2" class="section level4">
<h4>Volcano</h4>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T,label_size=5,adjusted = F)</code></pre>
<p><img src="figure/REV_REVTAX_DMSO.Rmd/volcano_3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="unique-2" class="section level3">
<h3>Unique</h3>
<pre class="r"><code>unique_pair(data_filt_Unique, conditions = conditions)</code></pre>
<div id="htmlwidget-afbb4563868722d3ca41" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-afbb4563868722d3ca41">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73"],["AKR7A2","AKT1","ARHGAP17","B4GAT1","BABAM1","BAZ1A","BNIP1","C5orf51","CIAO1","COG4","COX15","CRLF3","DCPS","DDX27","DDX49","DNAJC9","DNM1","DOCK6","FAM160B1","FAM57A","FANCA","FBXO21","FNBP1","FTL","GCAT","GFM2","HADH","IMPACT","IPO13","KIAA1033","KIF13A","MAGEA1","MEN1","METTL17","MIS18A","MYO9B","NARF","NEK7","OGFOD3","PANK3","PDZD8","PHAX","PICK1","PIGU","PPP1R18","PPP1R7","PPP2R5A","PPT1","PRKAR2B","PSTPIP2","RAD21","RALY","RBM39","RPP25L","RQCD1","RRN3","SF3A3","SLC18B1","SLC7A6OS","SMN1","STAG2","STAMBP","STK24","TAF5L","TBC1D10B","TCERG1","THAP11","TOE1","TRAPPC12","ULK3","USP54","XPO4","YIPF6"],[true,false,true,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,true,false,false,false,false,false,false,false,true,false,true,false,false,true,false,true,false,false,false,false,true,true,true,false,true,false,false,true,false,false,false,false,true,false,false,true,false,false,false,false,true,false,false,false,true,true,false,false,false,true,false,true,true],[false,true,false,true,true,true,true,true,true,true,true,true,true,true,true,false,true,false,true,true,false,true,true,true,true,true,true,true,false,true,false,true,true,false,true,false,true,true,true,true,false,false,false,true,false,true,true,false,true,true,true,true,false,true,true,false,true,true,true,true,false,true,true,true,false,false,true,true,true,false,true,false,false]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>unique_REVTAX<\/th>\n      <th>unique_DMSO<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cc-unique-2" class="section level3">
<h3>CC Unique</h3>
<pre class="r"><code>tab_unique &lt;- unique_pair(data_filt_Unique, conditions = conditions, table = T)

GO_name &lt;- tab_unique %&gt;% filter(unique_REVTAX) %&gt;% pull(name)

GO &lt;- gost(query = GO_name, organism = &quot;hsapiens&quot;, sources = &quot;GO:CC&quot;, ordered_query = F, significant = F)

gostplot(GO, capped = TRUE, interactive = TRUE)</code></pre>
<div id="htmlwidget-46c1628725fa50833c0c" style="width:960px;height:960px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-46c1628725fa50833c0c">{"x":{"data":[{"x":[21.7689018547267,164.016146211659,14.5721330896352,2.79964097389906,96.046663430239,96.4464839171885,96.4909084157385,96.5353329142884,96.5797574128384,96.6241819113883,96.6686064099383,96.7130309084882,96.7574554070382,96.8463044041381,97.6015208794872,103.77652617793,103.82095067648,95.0693244621401,94.9804754650402,94.9360509664903,94.4473814824409,65.9268534133745,66.1934004046742,66.2378249032242,66.4599473959739,66.5043718945239,66.5932208916237,104.398469157629,66.6376453901737,70.6802747582189,71.7020382248677,77.9214680218604,78.27686401026,89.2052906535471,91.781911569444,68.9921438133209,110.218078467672,114.882650815417,60.6847625844807,163.03880724356,163.571901226159,164.149419707309,164.460391197158,166.503918130456,168.547445063753,162.99438274501,168.591869562303,169.924604518802,169.969029017352,170.368849504301,173.256441910048,173.434139904248,177.921014257792,169.880180020252,110.662323453172,162.94995824646,161.128553805912,122.745787058757,122.790211557307,123.811975023956,124.744889493505,124.789313992055,126.210897945653,162.416864263861,141.093104959886,146.20192229313,150.733221145224,151.088617133624,153.220993064022,157.308046930617,161.084129307362,142.248141922184,56.9086802077352,56.0201902367362,51.8887118715911,19.9030729156289,20.1251954083786,20.3028934025784,20.3473179011283,20.4805913967782,20.5694403938781,19.1034319417298,20.9248363822777,21.4135058663271,21.457930364877,21.5467793619769,21.7244773561767,22.1687223416762,22.2575713387761,21.1913833735773,22.4352693329759,19.0590074431799,17.4153009968318,2.844065472449,4.93201690429654,5.6428088810957,6.44244985499476,6.70899684629444,6.88669484049423,18.7480359533302,8.08615630134282,12.6618796519874,14.3055860983355,15.0608025736846,15.1052270722345,15.4161985620842,15.8160190490337,12.6174551534375,182.141341620037,22.8350898199254,24.2566737735237,39.7608237674554,40.5160402428045,40.6493137384544,41.8931996978529,42.0264731935028,47.5351110136963,39.7163992689055,49.6230624455438,49.7119114426437,49.8451849382935,50.7780994078424,50.8225239063924,51.3556178889917,51.6221648802914,49.6674869440937,23.8124287880243,39.1833052863061,39.0056072921063,24.3899472691736,24.4343717677235,24.7453432575732,24.834192254673,25.8115312227719,26.3002007068213,39.1388807877562,26.5223231995711,28.4770011357687,29.2322176111179,37.2730518486584,38.2503908167572,38.2948153153072,38.5613623066068,26.8777191879706,199.289198060317],"y":[0.231562325661523,0.176537557600133,0.0826944068926654,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0,-0],"text":["GO:0005802 (255) <br> trans-Golgi network <br> 5.867e-01","GO:0098842 (3) <br> postsynaptic early endosome <br> 6.660e-01","GO:0005622 (14808) <br> intracellular anatomical structure <br> 8.266e-01","GO:0000138 (11) <br> Golgi trans cisterna <br> 1.000e+00","GO:0043202 (97) <br> lysosomal lumen <br> 1.000e+00","GO:0043226 (14027) <br> organelle <br> 1.000e+00","GO:0043227 (13201) <br> membrane-bounded organelle <br> 1.000e+00","GO:0043228 (5102) <br> non-membrane-bounded organelle <br> 1.000e+00","GO:0043229 (13117) <br> intracellular organelle <br> 1.000e+00","GO:0043230 (2265) <br> extracellular organelle <br> 1.000e+00","GO:0043231 (12046) <br> intracellular membrane-bounded organelle <br> 1.000e+00","GO:0043232 (5094) <br> intracellular non-membrane-bounded organelle <br> 1.000e+00","GO:0043233 (5579) <br> organelle lumen <br> 1.000e+00","GO:0043240 (14) <br> Fanconi anaemia nuclear complex <br> 1.000e+00","GO:0043296 (146) <br> apical junction complex <br> 1.000e+00","GO:0044232 (40) <br> organelle membrane contact site <br> 1.000e+00","GO:0044233 (20) <br> mitochondria-associated endoplasmic reticulum membrane <br> 1.000e+00","GO:0043025 (523) <br> neuronal cell body <br> 1.000e+00","GO:0043005 (1384) <br> neuron projection <br> 1.000e+00","GO:0042995 (2337) <br> cell projection <br> 1.000e+00","GO:0042734 (148) <br> presynaptic membrane <br> 1.000e+00","GO:0031967 (1244) <br> organelle envelope <br> 1.000e+00","GO:0031974 (5579) <br> membrane-enclosed lumen <br> 1.000e+00","GO:0031975 (1244) <br> envelope <br> 1.000e+00","GO:0031981 (4453) <br> nuclear lumen <br> 1.000e+00","GO:0031982 (4058) <br> vesicle <br> 1.000e+00","GO:0031984 (1764) <br> organelle subcompartment <br> 1.000e+00","GO:0044297 (595) <br> cell body <br> 1.000e+00","GO:0031985 (117) <br> Golgi cisterna <br> 1.000e+00","GO:0032588 (98) <br> trans-Golgi network membrane <br> 1.000e+00","GO:0032991 (5541) <br> protein-containing complex <br> 1.000e+00","GO:0034399 (132) <br> nuclear periphery <br> 1.000e+00","GO:0034451 (107) <br> centriolar satellite <br> 1.000e+00","GO:0036477 (875) <br> somatodendritic compartment <br> 1.000e+00","GO:0042175 (1154) <br> nuclear outer membrane-endoplasmic reticulum membrane network <br> 1.000e+00","GO:0032279 (345) <br> asymmetric synapse <br> 1.000e+00","GO:0045121 (339) <br> membrane raft <br> 1.000e+00","GO:0048471 (745) <br> perinuclear region of cytoplasm <br> 1.000e+00","GO:0031410 (2433) <br> cytoplasmic vesicle <br> 1.000e+00","GO:0098794 (641) <br> postsynapse <br> 1.000e+00","GO:0098827 (1137) <br> endoplasmic reticulum subcompartment <br> 1.000e+00","GO:0098845 (15) <br> postsynaptic endosome <br> 1.000e+00","GO:0098857 (339) <br> membrane microdomain <br> 1.000e+00","GO:0098984 (371) <br> neuron to neuron synapse <br> 1.000e+00","GO:0099080 (1325) <br> supramolecular complex <br> 1.000e+00","GO:0098793 (515) <br> presynapse <br> 1.000e+00","GO:0099081 (1005) <br> supramolecular polymer <br> 1.000e+00","GO:0099512 (997) <br> supramolecular fiber <br> 1.000e+00","GO:0099513 (766) <br> polymeric cytoskeletal fiber <br> 1.000e+00","GO:0099572 (363) <br> postsynaptic specialization <br> 1.000e+00","GO:0110165 (18814) <br> cellular anatomical entity <br> 1.000e+00","GO:0120025 (2234) <br> plasma membrane bounded cell projection <br> 1.000e+00","GO:0140513 (1236) <br> nuclear protein-containing complex <br> 1.000e+00","GO:0099503 (1027) <br> secretory vesicle <br> 1.000e+00","GO:0045202 (1348) <br> synapse <br> 1.000e+00","GO:0098791 (845) <br> Golgi apparatus subcompartment <br> 1.000e+00","GO:0098590 (1240) <br> plasma membrane region <br> 1.000e+00","GO:0065010 (2265) <br> extracellular membrane-bounded organelle <br> 1.000e+00","GO:0070013 (5579) <br> intracellular organelle lumen <br> 1.000e+00","GO:0070062 (2178) <br> extracellular exosome <br> 1.000e+00","GO:0070160 (132) <br> tight junction <br> 1.000e+00","GO:0070161 (838) <br> anchoring junction <br> 1.000e+00","GO:0070382 (228) <br> exocytic vesicle <br> 1.000e+00","GO:0098687 (348) <br> chromosomal region <br> 1.000e+00","GO:0071944 (6178) <br> cell periphery <br> 1.000e+00","GO:0097060 (384) <br> synaptic membrane <br> 1.000e+00","GO:0097431 (36) <br> mitotic spindle pole <br> 1.000e+00","GO:0097447 (643) <br> dendritic tree <br> 1.000e+00","GO:0097542 (47) <br> ciliary tip <br> 1.000e+00","GO:0097708 (2438) <br> intracellular vesicle <br> 1.000e+00","GO:0098588 (2165) <br> bounding membrane of organelle <br> 1.000e+00","GO:0072686 (161) <br> mitotic spindle <br> 1.000e+00","GO:0031224 (5873) <br> intrinsic component of membrane <br> 1.000e+00","GO:0031090 (3644) <br> organelle membrane <br> 1.000e+00","GO:0030666 (190) <br> endocytic vesicle membrane <br> 1.000e+00","GO:0005759 (482) <br> mitochondrial matrix <br> 1.000e+00","GO:0005764 (723) <br> lysosome <br> 1.000e+00","GO:0005768 (993) <br> endosome <br> 1.000e+00","GO:0005769 (387) <br> early endosome <br> 1.000e+00","GO:0005773 (818) <br> vacuole <br> 1.000e+00","GO:0005775 (174) <br> vacuolar lumen <br> 1.000e+00","GO:0005739 (1678) <br> mitochondrion <br> 1.000e+00","GO:0005783 (1995) <br> endoplasmic reticulum <br> 1.000e+00","GO:0005794 (1593) <br> Golgi apparatus <br> 1.000e+00","GO:0005795 (151) <br> Golgi stack <br> 1.000e+00","GO:0005797 (11) <br> Golgi medial cisterna <br> 1.000e+00","GO:0005801 (71) <br> cis-Golgi network <br> 1.000e+00","GO:0005813 (620) <br> centrosome <br> 1.000e+00","GO:0005815 (815) <br> microtubule organizing center <br> 1.000e+00","GO:0005789 (1132) <br> endoplasmic reticulum membrane <br> 1.000e+00","GO:0005819 (390) <br> spindle <br> 1.000e+00","GO:0005737 (11951) <br> cytoplasm <br> 1.000e+00","GO:0005694 (1795) <br> chromosome <br> 1.000e+00","GO:0000139 (725) <br> Golgi membrane <br> 1.000e+00","GO:0000323 (723) <br> lytic vacuole <br> 1.000e+00","GO:0000407 (33) <br> phagophore assembly site <br> 1.000e+00","GO:0000775 (200) <br> chromosome, centromeric region <br> 1.000e+00","GO:0000785 (1235) <br> chromatin <br> 1.000e+00","GO:0000793 (239) <br> condensed chromosome <br> 1.000e+00","GO:0005730 (955) <br> nucleolus <br> 1.000e+00","GO:0000922 (168) <br> spindle pole <br> 1.000e+00","GO:0005576 (4564) <br> extracellular region <br> 1.000e+00","GO:0005615 (3595) <br> extracellular space <br> 1.000e+00","GO:0005634 (7607) <br> nucleus <br> 1.000e+00","GO:0005635 (471) <br> nuclear envelope <br> 1.000e+00","GO:0005643 (85) <br> nuclear pore <br> 1.000e+00","GO:0005654 (4112) <br> nucleoplasm <br> 1.000e+00","GO:0005575 (18964) <br> cellular_component <br> 1.000e+00","GO:1903561 (2263) <br> extracellular vesicle <br> 1.000e+00","GO:0005829 (5361) <br> cytosol <br> 1.000e+00","GO:0005871 (52) <br> kinesin complex <br> 1.000e+00","GO:0016021 (5718) <br> integral component of membrane <br> 1.000e+00","GO:0016363 (110) <br> nuclear matrix <br> 1.000e+00","GO:0016459 (57) <br> myosin complex <br> 1.000e+00","GO:0016604 (816) <br> nuclear body <br> 1.000e+00","GO:0016607 (413) <br> nuclear speck <br> 1.000e+00","GO:0030054 (2107) <br> cell junction <br> 1.000e+00","GO:0016020 (9829) <br> membrane <br> 1.000e+00","GO:0030133 (416) <br> transport vesicle <br> 1.000e+00","GO:0030135 (296) <br> coated vesicle <br> 1.000e+00","GO:0030139 (335) <br> endocytic vesicle <br> 1.000e+00","GO:0030424 (661) <br> axon <br> 1.000e+00","GO:0030425 (641) <br> dendrite <br> 1.000e+00","GO:0030496 (197) <br> midbody <br> 1.000e+00","GO:0030659 (789) <br> cytoplasmic vesicle membrane <br> 1.000e+00","GO:0030134 (90) <br> COPII-coated ER to Golgi transport vesicle <br> 1.000e+00","GO:0005856 (2339) <br> cytoskeleton <br> 1.000e+00","GO:0015630 (1313) <br> microtubule cytoskeleton <br> 1.000e+00","GO:0015030 (57) <br> Cajal body <br> 1.000e+00","GO:0005874 (435) <br> microtubule <br> 1.000e+00","GO:0005875 (164) <br> microtubule associated complex <br> 1.000e+00","GO:0005884 (118) <br> actin filament <br> 1.000e+00","GO:0005886 (5680) <br> plasma membrane <br> 1.000e+00","GO:0005911 (496) <br> cell-cell junction <br> 1.000e+00","GO:0005923 (124) <br> bicellular tight junction <br> 1.000e+00","GO:0015629 (517) <br> actin cytoskeleton <br> 1.000e+00","GO:0005929 (674) <br> cilium <br> 1.000e+00","GO:0008021 (208) <br> synaptic vesicle <br> 1.000e+00","GO:0008278 (10) <br> cohesin complex <br> 1.000e+00","GO:0010008 (520) <br> endosome membrane <br> 1.000e+00","GO:0012505 (4651) <br> endomembrane system <br> 1.000e+00","GO:0012506 (1183) <br> vesicle membrane <br> 1.000e+00","GO:0014069 (339) <br> postsynaptic density <br> 1.000e+00","GO:0005938 (326) <br> cell cortex <br> 1.000e+00","GO:1990904 (694) <br> ribonucleoprotein complex <br> 1.000e+00"],"key":["GO:0005802","GO:0098842","GO:0005622","GO:0000138","GO:0043202","GO:0043226","GO:0043227","GO:0043228","GO:0043229","GO:0043230","GO:0043231","GO:0043232","GO:0043233","GO:0043240","GO:0043296","GO:0044232","GO:0044233","GO:0043025","GO:0043005","GO:0042995","GO:0042734","GO:0031967","GO:0031974","GO:0031975","GO:0031981","GO:0031982","GO:0031984","GO:0044297","GO:0031985","GO:0032588","GO:0032991","GO:0034399","GO:0034451","GO:0036477","GO:0042175","GO:0032279","GO:0045121","GO:0048471","GO:0031410","GO:0098794","GO:0098827","GO:0098845","GO:0098857","GO:0098984","GO:0099080","GO:0098793","GO:0099081","GO:0099512","GO:0099513","GO:0099572","GO:0110165","GO:0120025","GO:0140513","GO:0099503","GO:0045202","GO:0098791","GO:0098590","GO:0065010","GO:0070013","GO:0070062","GO:0070160","GO:0070161","GO:0070382","GO:0098687","GO:0071944","GO:0097060","GO:0097431","GO:0097447","GO:0097542","GO:0097708","GO:0098588","GO:0072686","GO:0031224","GO:0031090","GO:0030666","GO:0005759","GO:0005764","GO:0005768","GO:0005769","GO:0005773","GO:0005775","GO:0005739","GO:0005783","GO:0005794","GO:0005795","GO:0005797","GO:0005801","GO:0005813","GO:0005815","GO:0005789","GO:0005819","GO:0005737","GO:0005694","GO:0000139","GO:0000323","GO:0000407","GO:0000775","GO:0000785","GO:0000793","GO:0005730","GO:0000922","GO:0005576","GO:0005615","GO:0005634","GO:0005635","GO:0005643","GO:0005654","GO:0005575","GO:1903561","GO:0005829","GO:0005871","GO:0016021","GO:0016363","GO:0016459","GO:0016604","GO:0016607","GO:0030054","GO:0016020","GO:0030133","GO:0030135","GO:0030139","GO:0030424","GO:0030425","GO:0030496","GO:0030659","GO:0030134","GO:0005856","GO:0015630","GO:0015030","GO:0005874","GO:0005875","GO:0005884","GO:0005886","GO:0005911","GO:0005923","GO:0015629","GO:0005929","GO:0008021","GO:0008278","GO:0010008","GO:0012505","GO:0012506","GO:0014069","GO:0005938","GO:1990904"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":[0.3,0.3,0.3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"size":[17.2437966468348,3.77952755905512,22.4081708585755,11.0609050434185,15.6894167298696,22.3487316742076,22.2819265838077,21.2021048702491,22.2748860463327,20.2236577948149,22.1806846166035,21.2002670878108,21.306458533971,11.7079133647439,16.3704302048982,14.0604942317227,12.5780363166136,18.2915286949929,19.6006697950103,20.262439057463,16.3924582574199,19.4625361165237,21.306458533971,19.4625361165237,21.0420408511073,20.9319032594411,19.9105012002746,18.471747411219,16.0063448646879,15.7069743163751,21.2985006689267,16.2060073242099,15.8563338833562,18.9978928663639,19.3645306176304,17.6943181046261,17.6685694249791,18.780708494467,20.3121951102797,18.5747991925802,19.3450895902551,11.8835154767517,17.6685694249791,17.8004497523251,19.5443876860979,18.2698405744641,19.1824830340175,19.1718930128304,18.8184687078291,17.7686919814549,22.66858966672,20.2065501203874,19.4541405494408,19.2111400797646,19.5666446721524,18.9510480950724,19.45834566556,20.2236577948149,21.306458533971,20.1749886124466,16.2060073242099,18.9398570442927,17.0731210070953,17.7070074559992,21.424779176041,17.8504776184389,13.8492316899623,18.5790945833976,14.3757048606121,20.314728483431,20.167536948996,16.5279218843464,21.3661400858925,20.8034206606188,16.7902856082853,18.1762941579758,18.7398842287607,19.1665633385199,17.8617572510157,18.9073129736584,16.6515972027493,19.8471588366245,20.0654077275161,19.7810130013538,16.424878344691,11.0609050434185,15.1423062227101,18.5287925525793,18.9023567066574,19.3393116192235,17.8729408579862,22.1719034792789,19.9325208868513,18.7436511664074,18.7398842287607,13.6713605852676,16.8704746559872,19.4530869673257,17.1452418199174,19.1147370478491,16.595854753999,21.0711169787004,20.7871877759916,21.6637544197659,18.1435395978907,15.4609931999524,20.947620567131,22.6771653543307,20.2225616939645,21.2599973646214,14.5686097750412,21.3350829451696,15.902959317746,14.7408499257426,18.904011000437,17.9556523249751,20.1336939190228,21.9537631460362,17.9660653032093,17.4678634757917,17.6511220894254,18.6171082212776,18.5747991925802,16.846897981604,18.8585513950913,15.5604036761575,20.2634978990397,19.5326087803874,14.7408499257426,18.0301513248634,16.5574350412905,16.0205384683609,21.3273318832449,18.2168138835123,16.1029279696165,18.2752970806948,18.6438656665855,16.931460261082,10.7887526937734,18.2834385695774,21.093383158747,19.3969889362917,17.6685694249791,17.6110083411357,18.6839493329099],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData2142452e","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,200.044414535666],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":45.251350032552,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:CC"],"tickvals":[101.022207267833],"categoryorder":"array","categoryarray":["GO:CC"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":-0.0108706932337069,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis"},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","showSendToCloud":false},"source":"A","attrs":{"45d422f83019":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"45d42faf778b":{"x":{},"xend":{},"y":{},"yend":{}},"45d42b066d5a":{"x":{},"y":{}},"45d427d65324":{"yintercept":{}}},"cur_data":"45d422f83019","visdat":{"45d422f83019":["function (y) ","x"],"45d42faf778b":["function (y) ","x"],"45d42b066d5a":["function (y) ","x"],"45d427d65324":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData2142452e"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="unique-in-each-conditions" class="section level1">
<h1><strong>Unique in each conditions</strong></h1>
<pre class="r"><code>expdesign &lt;- read.table(here::here(&quot;data/expdesign.tsv&quot;), header = T, stringsAsFactors = F)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

data_filt &lt;- filter_missval(data_se, thr = 0)

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;%
  mutate(intensity = case_when(is.na(intensity) ~ 0, TRUE ~ as.numeric(1))) %&gt;%
  group_by(rowname) %&gt;%
  mutate(unique = if_else(sum(intensity) == 3, T, F)) %&gt;%
  ungroup() %&gt;%
  filter(unique) %&gt;% 
  mutate(condition = gsub(pattern = &quot;..$&quot;, replacement = &quot;&quot;,x = samples)) %&gt;% 
  group_by(rowname, condition) %&gt;% 
  mutate(unique_2 = if_else(sum(intensity) == 0, F, T)) %&gt;% 
  ungroup() %&gt;% 
  mutate(unique_3 = case_when(unique_2 ~ condition)) %&gt;% 
  select(rowname, samples, intensity, unique_3) %&gt;%
  group_by(rowname) %&gt;% 
  fill(unique_3, .direction = &quot;downup&quot;) %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(id_cols = c(rowname, unique_3) ,names_from = samples, values_from = intensity) %&gt;% 
  dplyr::rename(Unique = unique_3, name = rowname) %&gt;% 
  select(1:2) %&gt;%
  as.data.frame() %&gt;% 
    DT::datatable(options = list(
      columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:2)),
      pageLength = 10))</code></pre>
<div id="htmlwidget-454749642ed2cefeb54d" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-454749642ed2cefeb54d">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23"],["ARHGAP17","CIAO1","COG4","IMPACT","IPO13","ISOC1","KIAA1033","KIF13A","MPO","NARF","OGFOD3","PICK1","PIGU","PPP1R7","PRKAR2B","RALY","RRN3","SF3A3","SMN1","STAMBP","STK24","USP54","YIPF6"],["REVTAX","DMSO","DMSO","DMSO","REVTAX","REV.","DMSO","REVTAX","REV.","DMSO","DMSO","REVTAX","DMSO","DMSO","DMSO","DMSO","REVTAX","DMSO","DMSO","DMSO","DMSO","DMSO","REVTAX"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>name<\/th>\n      <th>Unique<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<div id="section" class="section level2 unnumbered">
<h2></h2>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] openxlsx_4.2.4              gprofiler2_0.2.0           
 [3] naniar_0.6.1                visdat_0.5.3               
 [5] ggrepel_0.9.1               UpSetR_1.4.0               
 [7] ggstatsplot_0.8.0           datapasta_3.1.0            
 [9] here_1.0.1                  patchwork_1.1.1            
[11] stringr_1.4.0               DT_0.18                    
[13] enrichR_3.0                 ggplot2_3.3.5              
[15] tibble_3.1.2                preprocessCore_1.52.1      
[17] SummarizedExperiment_1.20.0 Biobase_2.50.0             
[19] GenomicRanges_1.42.0        GenomeInfoDb_1.26.2        
[21] IRanges_2.24.1              S4Vectors_0.28.1           
[23] BiocGenerics_0.36.0         MatrixGenerics_1.2.1       
[25] matrixStats_0.59.0          DEP_1.12.0                 
[27] tidyr_1.1.3                 dplyr_1.0.7                
[29] MBQN_2.2.0                  workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] utf8_1.2.1                shinydashboard_0.7.1     
  [3] gmm_1.6-6                 tidyselect_1.1.1         
  [5] RSQLite_2.2.7             htmlwidgets_1.5.3        
  [7] grid_4.0.3                BiocParallel_1.24.1      
  [9] gmp_0.6-2                 norm_1.0-9.5             
 [11] munsell_0.5.0             codetools_0.2-16         
 [13] effectsize_0.4.5          withr_2.4.2              
 [15] colorspace_2.0-2          highr_0.9                
 [17] knitr_1.33                ipmisc_6.0.2             
 [19] ggsignif_0.6.2            mzID_1.28.0              
 [21] labeling_0.4.2            git2r_0.28.0             
 [23] GenomeInfoDbData_1.2.4    farver_2.1.0             
 [25] bit64_4.0.5               rprojroot_2.0.2          
 [27] coda_0.19-4               vctrs_0.3.8              
 [29] generics_0.1.0            xfun_0.24                
 [31] BiocFileCache_1.14.0      BWStest_0.2.2            
 [33] R6_2.5.0                  doParallel_1.0.16        
 [35] clue_0.3-59               BayesFactor_0.9.12-4.2   
 [37] bitops_1.0-7              cachem_1.0.5             
 [39] reshape_0.8.8             DelayedArray_0.16.2      
 [41] assertthat_0.2.1          promises_1.2.0.1         
 [43] scales_1.1.1              gtable_0.3.0             
 [45] Cairo_1.5-12.2            affy_1.68.0              
 [47] multcompView_0.1-8        sandwich_3.0-1           
 [49] rlang_0.4.11              MatrixModels_0.5-0       
 [51] zeallot_0.1.0             mzR_2.24.1               
 [53] GlobalOptions_0.1.2       PMCMRplus_1.9.0          
 [55] lazyeval_0.2.2            impute_1.64.0            
 [57] BiocManager_1.30.16       yaml_2.2.1               
 [59] crosstalk_1.1.1           httpuv_1.6.1             
 [61] tools_4.0.3               affyio_1.60.0            
 [63] ellipsis_0.3.2            jquerylib_0.1.4          
 [65] RColorBrewer_1.1-2        WRS2_1.1-2               
 [67] MSnbase_2.15.7            Rcpp_1.0.6               
 [69] plyr_1.8.6                zlibbioc_1.36.0          
 [71] purrr_0.3.4               RCurl_1.98-1.3           
 [73] pbapply_1.4-3             GetoptLong_1.0.5         
 [75] correlation_0.6.1         zoo_1.8-9                
 [77] cluster_2.1.0             fs_1.5.0                 
 [79] magrittr_2.0.1            data.table_1.14.0        
 [81] circlize_0.4.13           pcaMethods_1.82.0        
 [83] mvtnorm_1.1-2             whisker_0.4              
 [85] ProtGenerics_1.22.0       hms_1.1.0                
 [87] mime_0.11                 evaluate_0.14            
 [89] xtable_1.8-4              XML_3.99-0.6             
 [91] pairwiseComparisons_3.1.6 gridExtra_2.3            
 [93] shape_1.4.6               compiler_4.0.3           
 [95] ncdf4_1.17                crayon_1.4.1             
 [97] htmltools_0.5.1.1         mc2d_0.1-21              
 [99] later_1.2.0               DBI_1.1.1                
[101] SuppDists_1.1-9.5         kSamples_1.2-9           
[103] dbplyr_2.1.1              ComplexHeatmap_2.6.2     
[105] MASS_7.3-53               tmvtnorm_1.4-10          
[107] rappdirs_0.3.3            Matrix_1.2-18            
[109] readr_1.4.0               vsn_3.58.0               
[111] imputeLCMD_2.0            insight_0.14.2           
[113] pkgconfig_2.0.3           statsExpressions_1.1.0   
[115] plotly_4.9.4.1            MALDIquant_1.19.3        
[117] paletteer_1.3.0           foreach_1.5.1            
[119] bslib_0.2.5.1             XVector_0.30.0           
[121] digest_0.6.27             parameters_0.14.0        
[123] rmarkdown_2.9             curl_4.3.2               
[125] shiny_1.6.0               gtools_3.9.2             
[127] rjson_0.2.20              lifecycle_1.0.0          
[129] jsonlite_1.7.2            viridisLite_0.4.0        
[131] limma_3.46.0              fansi_0.5.0              
[133] pillar_1.6.1              lattice_0.20-41          
[135] fastmap_1.1.0             httr_1.4.2               
[137] glue_1.4.2                zip_2.2.0                
[139] bayestestR_0.10.0         png_0.1-7                
[141] iterators_1.0.13          bit_4.0.4                
[143] stringi_1.6.2             sass_0.4.0               
[145] performance_0.7.2         rematch2_2.1.2           
[147] blob_1.2.1                memoise_2.0.0            
[149] Rmpfr_0.8-4              </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
